blockbase/sass/base/_mixins.scss

```diff
@@ -12,3 +12,19 @@
 	width: 1px;
 	word-wrap: normal !important;
 }
+
+@mixin post-meta-icon {
+	display: flex;
+	margin-right: calc(2 * var(--wp--custom--margin--baseline) );
+	&:before {
+		align-self: center;
+		content: '';
+		display: inline-block;
+		margin-right: calc(0.5 * var(--wp--custom--margin--baseline) );
+		height: 16px;
+		width: 16px;
+		mask-size: contain;
+		mask-repeat: no-repeat;
```

Comment:
@kjellr The issue you are seeing with the icons, I fixed it with this line, I'm not sure why you are not seeing it, is this line being applied on your test site?

Reply:
It's not — looks like [safari requires a `-webkit` prefix](https://caniuse.com/mdn-css_properties_mask-size).

Reply:
I thought the sass compiler took care of that! I'll add it, thank you for checking!